node/libs/roles/src/attester/messages/l1_batch.rs

```diff
@@ -1,22 +1,25 @@
-use crate::validator::{self};
+use crate::{
+    attester::{self, AggregateSignature},
+    validator::Genesis,
+};
 
-use super::{Genesis, Signed};
+use super::{SignedBatchMsg, Signers};
 
 /// A message to send by validators to the gossip network.
 /// It contains the validators signature to sign the block batches to be sent to L1.
-#[derive(Debug, Clone, Eq, PartialEq, Default)]
-pub struct L1BatchMsg;
+#[derive(Clone, Debug, PartialEq, Eq, Hash, Default)]
+pub struct L1Batch;
```

Comment:
I think we can already make this contain a Vec<u8>. That's what a L1Batch will be most likely.

Reply:
I thought that we want to include just a hash/other digest of the L1Batch of some form, no? (it doesn't change the fact that we will keep just a slice of bytes anyway, and let the user define the content). Also we will need the L1Batch number.

Reply:
True, we need the number. Would you prefer that we strongly type the hash? I guess if we have multiple fields then it makes sense. With just one field it looks weird.

Reply:
Given that we'll also have batch syncing later on, maybe we should already name this L1BatchHeader?